File: scheduled/src/main/scala/com/stulsoft/pspring/scheduled/SomeComponent.scala
/*
 * Copyright (c) 2018. Yuriy Stul
 */

package com.stulsoft.pspring.scheduled

import com.typesafe.scalalogging.LazyLogging
import org.springframework.stereotype.Component

/**
  * @author Yuriy Stul
  */
@Component
class SomeComponent extends LazyLogging {
  def work(): Unit = {
    logger.info("==>work")
    logger.info("<==work")
  }
}

File: scheduled-java/src/main/java/com/stulsoft/pspring/scheduledj/SomeComponent.java
/*
 * Copyright (c) 2018. Yuriy Stul
 */

package com.stulsoft.pspring.scheduledj;

import org.slf4j.Logger;
import org.slf4j.LoggerFactory;
import org.springframework.stereotype.Component;

/**
 * @author Yuriy Stul
 */
@Component
public class SomeComponent {
    private static final Logger logger = LoggerFactory.getLogger(SomeComponent.class);

    public void work() {
        logger.info("==>work");
        logger.info("<==work");
    }
}
